pages/index/_learn-more.html

```diff
@@ -8,33 +8,33 @@
       style="position: absolute; top: -190px; left: 166px;"
     >
     </canvas>
-    <img src="/assets/img/infrastructure-design-your-stack.png" width="379px" style="margin-left: 3px;" />
+    <img src="/assets/img/infrastructure-cube-icons/infrastructure-design-your-stack.png" width="379px" style="margin-left: 3px;" />
   </div>
   <div class="col-xs-12 col-sm-7 col-md-6 col-description">
-    <h2>Design your entire tech stack.</h2>
-    <p>Select the infrastructure you want from our library of battle-tested, reusable infrastructure code. Docker or Packer? MySQL or Postgres? Jenkins or CircleCI? You tell us, and we will deploy it into your AWS account and give you 100% of the code, all in about a day.</p>
-    <p><a href="/how-it-works" class="btn btn-primary" role="button">See how it works</a></p>
+    <h2>Get access to the Infrastructure as Code Library.</h2>
+    <p>Build your infrastructure on top of a collection of reusable, battle-tested infrastructure code written in Terraform, Go, Python, and Bash that has been proven in production at dozens of companies and is maintained and supported by DevOps experts.</p>
+    <p><a href="/infrastructure-as-code-library" class="btn btn-primary" role="button" ga-on="click" ga-event-category="home-learn-more" ga-event-action="browse-infra-as-code-library">Browse the Infrastructure as Code Library</a></p>
```

Comment:
"Browse the Infrastructure as Code Library" is quite a mouthful for a button. Perhaps "Browse the Library"?

Reply:
Haha, yea, that's a bit better: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/23/commits/e628daeebdd9eadc2f50f08575472050acbd5562